db/migrate/20230204044855_create_menu_items.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMenuItems < ActiveRecord::Migration[5.2]
+  def change
+    create_table :menu_items do |t|
+      t.belongs_to :restaurant, foreign_key: true
+      t.string :name
+      t.string :description
+      t.string :allergens
+      t.string :category
+      t.string :image
+      t.string :price
```

Comment:
Price should be a float rather than a string

Reply:
Also updated, thanks!